docs/reference-guides/rancher-security/hardening-guides/rke2-hardening-guide/rke2-self-assessment-guide-with-cis-v1.23-k8s-v1.23.md

```diff
@@ -2814,7 +2814,7 @@ for every namespace which contains user workloads.
 ### 5.2.2 Minimize the admission of privileged containers (Manual)
 
 
-**Result:** fail
+**Result:** pass
```

Comment:
Is this upgrade based on the result of the latest scan?

Reply:
Yes.